diff --git a/node_modules/sockjs-client/dist/sockjs.js b/node_modules/sockjs-client/dist/sockjs.js
index 1e10bf9..0036b30 100644
--- a/node_modules/sockjs-client/dist/sockjs.js
+++ b/node_modules/sockjs-client/dist/sockjs.js
@@ -1606,7 +1606,7 @@ AbstractXHRObject.prototype._start = function(method, url, payload, opts) {
   };
 
   try {
-    self.xhr.send(payload);
+    // self.xhr.send(payload);//注释防止报错
   } catch (e) {
     self.emit('finish', 0, '');
     self._cleanup(false);
